content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,151 @@
+   :align: center
+   :alt: The Amazon Accounts link beneath the Amazon Sync settings in Odoo Sales.
+
+Doing so reveals a separate :guilabel:`Amazon Accounts` page. From here, click :guilabel:`New` to
+create and link a new Amazon account.
+
+On the blank :guilabel:`Amazon Account` form page, start by choosing a name for the account (e.g.
+`American Marketplace`). Then, in the :guilabel:`Credentials` tab, select the marketplace on which
+the seller account was initially created from the :guilabel:`Home Marketplace` drop-down menu.
+
+.. image:: setup/amazon-accounts-form-page.png
+   :align: center
+   :alt: A typical Amazon Account form page in the Odoo Sales application.
+
+After saving, the field in the :guilabel:`Credentials` tab is replaced by a :guilabel:`Link with
+Amazon` button.
 
-#. Choose a meaningful name for your account (e.g. `Europe`), and select your :guilabel:`Sign-up
-   Marketplace`. This is the original sign-up marketplace of your seller account (e.g. if you
-   created your account on Amazon Germany, your sign-up marketplace will be `amazon.de`)
+.. image:: setup/amazon-accounts-form-link-button.png
+   :align: center
+   :alt: A typical Amazon Account form page and Link with Amazon button in Odoo Sales.
 
-#. Upon saving, a button to :guilabel:`link with Amazon` appears. Click on it to be redirected to
-   the Amazon login page, or directly to the consent page if you are already logged in. There,
-   confirm that you want Amazon to give Odoo access to your account and related data.
+Clicking that button redirects to either the Amazon login page, or directly to the required consent
+page, if the user is already logged in to Amazon.
 
-#. Amazon redirects you to Odoo, with your account registered.
+On the login page, log in to the desired Amazon seller account.
 
-Once the account is registered, the marketplaces available to this account are synchronized and
-listed under the :guilabel:`Marketplaces` tab. If you wish, you can remove some items from the list
-of synchronized marketplaces to disable their synchronization.
+On the consent page, confirm that Amazon is allowed to give Odoo access to the account and related
+data.
+
+Upon confirmation, Amazon returns the user to Odoo, and the account has been registered.
+
+With the Amazon account successfully registered, the marketplaces available to this specific account
+are synchronized with Odoo and listed under the :guilabel:`Marketplaces` tab.
+
+If desired, remove items from the list of synchronized marketplaces to disable synchronization.
 
 Match database products in Amazon
 =================================
 
-When an Amazon order is synchronized, up to three sales order items are created in Odoo for each
-product sold on Amazon: one for the marketplace product, one for the shipping charges (if any) and
-one for the gift wrapping charges (if any).
+When an Amazon order is synchronized, up to three line items are created on the sales order in Odoo.
```

Comment:
Q: does this happen automatically via the **Connector** or is there a manual sync that can be performed? It's not clear *how* orders are synchronized...